Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -106,6 +106,23 @@
     <Compile Include="**\*.tizen.cs" />
     <Compile Include="**\*.tizen.*.cs" />
   </ItemGroup>
+  <ItemGroup Condition=" $(TargetFramework.StartsWith('net461')) ">
+    <Compile Include="**\*.win32.*.cs" />
+    <Compile Include="**\*.win32.cs" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
```

Comment:
Taking a dependency on Newtonsoft.Json in Essentials will make it unusable in some platforms such as WatchOS because the AOT generates a massive amount of code xamarin/xamarin-macios#7250

Since only a limited number of types are supported in Preferences (bool, double, int, float, long, string, and DateTime) could you manually convert them to/from strings?